app/models/education_experience.rb

```diff
@@ -0,0 +1,25 @@
+class EducationExperience < ApplicationRecord
+  belongs_to :user
+
+  validates_presence_of :institution, :course, :start_date, :end_date
+
+  validates_comparison_of :start_date,
+    message: lambda { |education_experience, other|
+      I18n.t(
+        "activerecord.errors.models.education_experience.attributes.start_date.greater_than_current",
+        date: I18n.l(Date.current)
+      )
+    },
+    less_than: Date.current,
+    allow_nil: true
+
+  validates_comparison_of :end_date,
+    message: lambda { |education_experience, other|
+      I18n.t(
+        "activerecord.errors.models.education_experience.attributes.end_date.less_than_start_date",
+        date: I18n.l(education_experience.start_date)
+      )
```

Comment:
active record allows you to customize the message without having to pass here

Reply:
done